package/src/custom/Extension/modules/Contacts/Ext/Vardefs/pr_professors_contacts_Contacts.php

```diff
@@ -8,5 +8,5 @@
   'module' => 'PR_Professors',
   'bean_name' => 'PR_Professors',
   'vname' => 'LBL_PR_PROFESSORS_CONTACTS_FROM_PR_PROFESSORS_TITLE',
-  'id_name' => 'pr_professors_contactspr_professors_ida',
+  'id_name' => 'pr_prof_contactspr_prof_ida',
```

Comment:
I assume you are shortening index names to be under 30 characters? Is that the crux of the fix?